content/millennium/r4/base/workflow/appointment.md

```diff
@@ -51,11 +51,26 @@ The following fields are returned if valued:
 * [Comment](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.comment){:target="_blank"}
 * [Patient Instruction](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.patientInstruction){:target="_blank"}
 * [Requested period](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.requestedPeriod){:target="_blank"}
+* [Extensions](#extensions){:target="_blank"}
```

Comment:
Typically we list the supported extensions to make it something like "Extensions including is cancelable and is reschedulable"

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/921/commits/555aa76529c48bbe9ff8c9b16cf318de397a0ecb#diff-7d2b725b264b9a388dc5094d598e2a6e2ea109e548ff76f75902b8ae9ce589f3R54